modules/google-maps/src/google-maps-overlay.js

```diff
@@ -81,6 +84,7 @@ export default class GoogleMapsOverlay {
 
   /* Private API */
   _createOverlay(map) {
+    const {interleaved} = this.props;
```

Comment:
```suggestion
    const {interleaved = true} = this.props;
```

And you can drop the mutation in the constructor.

Reply:
As I also read the value out in utils.js I wanted it to be defaulted in one place. I've removed the mutation.